api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2296,8 +2296,113 @@ message ExportUpgradeWindowsResponse {
   string SystemdUnitSchedule = 3;
 }
 
+// AssistantMessageRequest is a request to the assistant service.
```

Comment:
I feel like this would have made sense as a new GRPC service instead of putting it in the auth service, but I imagine that's too big of a change to make now.

Reply:
Not "too big change". Moved